dockstore-webservice/src/main/java/io/dockstore/webservice/core/User.java

```diff
@@ -80,25 +85,11 @@
     @ApiModelProperty(value = "Indicates whether this user is an admin", required = true, position = 2)
     private boolean isAdmin;
 
-    @Column
-    @ApiModelProperty(value = "Company of user", position = 3)
-    private String company;
-
-    @Column
-    @ApiModelProperty(value = "Bio of user", position = 4)
-    private String bio;
-
-    @Column
-    @ApiModelProperty(value = "Location of user", position = 5)
-    private String location;
-
-    @Column
-    @ApiModelProperty(value = "Email of user", position = 6)
-    private String email;
-
-    @Column
-    @ApiModelProperty(value = "URL of user avatar on Github.", position = 7)
-    private String avatarUrl;
+    @ElementCollection(targetClass = Profile.class, fetch = FetchType.EAGER)
+    @JoinTable(name = "user_profile", joinColumns = @JoinColumn(name = "id"), uniqueConstraints = @UniqueConstraint(columnNames = {"id", "token_type"}))
+    @MapKeyColumn(name = "token_type", columnDefinition = "text")
+    @ApiModelProperty(value = "Profile information of the user attained from 3rd party sites (GitHub, Google, etc)")
+    private Map<String, Profile> userProfile = new HashMap<>();
```

Comment:
:+1: on the map. 

Let's try lazy rather than eager here, this seems like a good opportunity to start weaning ourselves off of eager loaded collections due to problems like 
https://hibernate.atlassian.net/browse/HHH-8776 and performance in general. 

Note that you may need to create a new endpoint to retrieve, say, the avatars for the multiple users of a starred tool.